### DISCUSSION TITLE
Why doesn't file tracking work?

### DISCUSSION THREAD

**Discussion by karlosvas** [community/community#66859] [Other Feature Feedback, Questions, & Ideas]
Somebody could help me?
I don't know why the changes are not automatically seen when saving in astro, the same thing happens to me with nodemon, I save and the changes are not shown on the page, I have to restart again with pnpm run dev.
Could it be some Visual Studio setting to track files?

**Comment by subidit** [comment]
Try 
`pnpm astro dev`

**Reply by karlosvas** [reply]
Same thing, it doesn't track me, thanks anyway 🥲
